x-pack/plugins/stack_connectors/kibana.jsonc

```diff
@@ -13,6 +13,7 @@
     "requiredPlugins": [
       "actions",
       "esUiShared",
+      "kibanaReact",
```

Comment:
Do we need this plugin?

Reply:
Tried to recall why I added it without success. Seems I do not need it. Removed.

Reply:
ERROR in ./public/connector_types/email/email_params.tsx
--
  | │          Module not found: Error: import [@kbn/kibana-react-plugin/public] references a public export of the [kibanaReact] bundle, but that bundle is not in the "requiredPlugins" or "requiredBundles" list in the plugin manifest [/dev/shm/builds/kb-n2-16-spot-46e986621280fcc4/elastic/kibana-pull-request/kibana/x-pack/plugins/stack_connectors/kibana.jsonc]
  | │           @ ./public/connector_types/email/email_params.tsx 13:0-60 35:6-15
  | │           @ ./public/connector_types/email/email.tsx
  | │           @ ./public/connector_types/email/index.ts
  | │           @ ./public/connector_types/index.ts
  | │           @ ./public/plugin.ts
  | │           @ ./public/index.ts

Added back.